chart/k8gb/values.yaml

```diff
@@ -10,6 +10,7 @@ k8gb:
   # image tag defaults to Chart.AppVersion, see Chart.yaml
   # but can be overrided with imageTag key
   # imageTag:
+  hookImage: bitnami/kubectl: 1.21.1
```

Comment:
should be `bitnami/kubectl:1.21.1` , without a space. Otherwise yaml will treat the colon+space as a standard separator

Reply:
@ytsarev nice catch. Done